Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -243,6 +243,154 @@ def __len__(self):
 		assert len(self.glyphOrder) == len(self.glyphs)
 		return len(self.glyphs)
 
+	def getPhantomPoints(self, glyphName, ttFont, defaultVerticalOrigin=None):
+		"""Compute the four "phantom points" for the given glyph from its bounding box
+		and the horizontal and vertical advance widths and sidebearings stored in the
+		ttFont's "hmtx" and "vmtx" tables.
+
+		If the ttFont doesn't contain a "vmtx" table, the hhea.ascent is used as the
+		vertical origin, and the head.unitsPerEm as the vertical advance.
+
+		The "defaultVerticalOrigin" (Optional[int]) is used when the ttFont contains
+		neither a "vmtx" nor an "hhea" table.
+
+		https://docs.microsoft.com/en-us/typography/opentype/spec/tt_instructing_glyphs#phantoms
+		"""
+		glyph = self[glyphName]
+		assert glyphName in ttFont["hmtx"].metrics, ttFont["hmtx"].metrics
```

Comment:
Looks like ttFont["hmtx"].metrics is being checked twice.

Reply:
I don't think so. The optional expression after the comma in an `assert` statement is printed (for debugging purposes) when the assertion fails.